routes/api.php

```diff
@@ -18,27 +19,32 @@
     return $request->user();
 });
 
-Route::resource('election', 'ElectionController', ['only' => ['index', 'store', 'show', 'update', 'destroy']]);
-Route::post('election/{election_id}/batchvote', 'ElectionController@batchvote')->name('election.batchvote');
-Route::get('election/{election_id}/batchvote', 'ElectionController@batchvote_view')->name('election.batchvote_view');
-Route::get('election/{election_id}/get_ready', 'ElectionController@get_ready')->name('election.get_ready');
-Route::post('election/{election_id}/set_ready', 'ElectionController@set_ready')->name('election.set_ready');
-Route::get('election/{election_id}/voter_stats', 'ElectionController@voter_stats')->name('election.voter_stats');
-Route::get('election/{election_id}/voter_details', 'ElectionController@voter_details')->name('election.voter_details');
-Route::post('election/{election_id}/batch_candidates', 'ElectionController@batch_candidates')->name('election.batch_candidates');
-Route::get('election/{election_id}/batch_candidates', 'ElectionController@batch_candidates_view')->name('election.batch_candidates_view');
+Route::group(['middleware' => 'auth:api'], function (Router $api) {
```

Comment:
Aha! I didn't know how to apply middleware to multiple routes at a time. Thanks for teaching me!
@neckberg @tylerharter if you want to learn more:
https://laravel.com/docs/5.6/routing#route-group-middleware